### PR TITLE
zh-tw: link to Node.js Foundation calendar

### PR DESCRIPTION
**Pull request by suensummit**
Ref #1859

### CODE REVIEW

File: locale/zh-tw/get-involved/index.md [hunk 1]
```diff
@@ -9,9 +9,10 @@ Node.js 是個包容的大家庭，我們鼓勵用戶一展長才。若你想[
 
 ## 社群討論
 
-- [GitHub issues 清單](https://github.com/nodejs/node/issues)是討論 Node.js 核心功能的好地方。
+- [GitHub issues 清單](https://github.com/nodejs/node/issues) 是討論 Node.js 核心功能的好地方。
 - 關於 Node.js 開發的即時對話請使用 [IRC 用戶端](http://en.wikipedia.org/wiki/Comparison_of_Internet_Relay_Chat_clients) 或 [freenode 網頁聊天室](http://webchat.freenode.net/?channels=node.js) 連線至 `irc.freenode.net` 中的 `#node.js` 頻道。
-- Node.js 官方的 Twitter 帳號是 [nodejs](https://twitter.com/nodejs).
+- Node.js 官方的 Twitter 帳號是 [nodejs](https://twitter.com/nodejs)。
+- [Node.js Foundation calendar](https://nodejs.org/calendar) 是公開團隊會議的日程表。
```
Comment (osk2) [inline locale/zh-tw/get-involved/index.md]:
```suggestion
- [Node.js Foundation 行事曆](https://nodejs.org/calendar) 中列出了所有的公開團隊會議。
```

Comment (osk2) [reply locale/zh-tw/get-involved/index.md]: 句子唸起來不怎麼順暢，看看這樣有沒有比較好

Comment (suensummit) [reply locale/zh-tw/get-involved/index.md]: LGTM 👍

**Review by osk2 — APPROVED** [review]

**Comment by ghost** [comment]
@osk2：I feel very interested and strange：Since this is a PR by suensummit, so how can you do a direct submit following his? Don't you need to create a new PR, or github has the new function?!

**Comment by osk2** [comment]
I just push commit to his branch :)

**Comment by ghost** [comment]
@osk2：

Do u mean:

1) Use "git remote add" to add suensummit's remote url origin.
2) Then git push to his?

You can do submits to other man's proj without being allowed? :open_mouth:

**Comment by osk2** [comment]
Please take a look at [this well explained article](https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/)
on GitHub Help

**Comment by ghost** [comment]
Thanks!

**Pull request merged**